src/cmd-buildextend-metal

```diff
@@ -67,10 +67,14 @@ if [ $# -ne 0 ]; then
 fi
 
 # default to both BIOS and UEFI
-if [ -z "${BIOS}" ] && [ -z "${UEFI}" ] && [ -z "${DASD}" ]; then
+if [ -z "${BIOS}" ] && [ -z "${UEFI}" ] && [ -z "${DASD}" ] && [ "${arch}" != "s390x"]; then
     BIOS=1
     UEFI=1
```

Comment:
just move the arch check inside? 
`if [ "$arch" = "s390x" ]; then
    DASD=1
else
    UEFI=1
`